spec/sh-options.test.sh

```diff
@@ -320,6 +320,92 @@ f.o
 .oo
 ## END
 
+#### noclobber on >>
+rm -f $TMP/no-clobber
```

Comment:
BTW the spec test framework now defaults to creating a new dir for each test case, and cd's to that dir

So you no longer have to use $TMP, it is OK to just use `touch no-clobber`